pkgs/tools/security/neopg/default.nix

```diff
@@ -12,13 +12,13 @@
 
 stdenv.mkDerivation rec {
   name = "neopg-${version}";
-  version = "0.0.4";
+  version = "0.0.5.2018-11-10";
```

Comment:
Version number should be something like `0.0.5pre20181110` or similar, to work properly with [the version comparison algorithm](https://nixos.org/nix/manual/#ssec-version-comparisons) when `0.0.5` will actually be out.

Reply:
Actually it should even be `0.0.6pre2018-11-10` or similar, as `0.0.5` is dated from before the pointed-to commit https://github.com/das-labor/neopg/tree/v0.0.5

Reply:
I agree! Changed the commit message and opened an issue in https://github.com/das-labor/neopg/issues/97